### PR TITLE
Updated all libraries to latest version.

### PR DESCRIPTION
**Pull request by Tc2r1**
## Description

Updated all libraries to latest version.
Tested code to ensure compatibility. 

Hacktoberfest

## Issue Resolved

Link to the issue that is resolved by this pull request goes here e.g. 'Fixes #10'.

## Screenshots 

Provide some screenshots of the changes made (if applicable).

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -9,12 +9,12 @@ plugins {
...
 
 android {
-    compileSdk 31
+    compileSdk 33
 
     defaultConfig {
         applicationId "com.pec_acm.moviedroid"
         minSdk 21
-        targetSdk 31
+        targetSdk 33
```
Comment (ken1000minus7) [inline app/build.gradle:17]: Please remove the changes to `compileSdk` and `targetSdk`

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (Tc2r1) [reply app/build.gradle:17]: @ken1000minus7 if these are removed the libraries will have to be reverted as they have a min version that is higher than 31. This compiles just fine and is most likely the best direction to go with keeping your project up to date. Is there a problem ?

Comment (ken1000minus7) [reply app/build.gradle:17]: Oh I see, then maybe we can update them as well🤔

Comment (Tc2r1) [reply app/build.gradle:17]: @ken1000minus7 I'm 10 years a senior And Dev and also the one who just did a similar update at my company, I can assure you the updates were compatible (Thank God) however check out the branch yourself and run the code if you need to see I understand.

Comment (ken1000minus7) [reply app/build.gradle:17]: Interesting 😯, thanks for the advice😄

**Review by ken1000minus7 — APPROVED** [review]

**Pull request merged**